src/JoomlaBrowser.php

```diff
@@ -678,6 +678,11 @@ public function clickToolbarButton($button)
 		$I = $this;
 		$input = strtolower($button);
 
+		$element = $this->webDriver->findElements(WebDriverBy::linkText("Toolbar"));
+		if(!empty($element) && $element[0]->isDisplayed())
+		{
+			$I->click('Toolbar');
+		}
```

Comment:
Note from Puneet:
These lines are to handle the case in which we change the resolution to tab/mobile
in that case we can't directly access toolbar buttons like 'New', 'Edit'
we first need to click on 'Toolbar' button and then click the respective button
now to keep the function generic such that it works on all screen resolutions
I have added lines to check if Toolbar button is present, if it is present then we need to first click that button and then click Toolbar buttons like 'New', 'Edit'

​[12:04]